go.mod

```diff
@@ -14,21 +15,24 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pierrec/lz4 v2.3.0+incompatible // indirect
 	github.com/robfig/cron v1.2.0
-	github.com/terraform-providers/terraform-provider-aws v2.32.0+incompatible
-	github.com/terraform-providers/terraform-provider-google v2.17.0+incompatible
-	github.com/terraform-providers/terraform-provider-random v2.2.1+incompatible // indirect
+	github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20191010190908-1261a98537f2
+	github.com/terraform-providers/terraform-provider-google v1.20.1-0.20191008212436-363f2d283518
+	github.com/terraform-providers/terraform-provider-random v1.3.2-0.20190925210718-83518d96ae4f // indirect
 	github.com/ulikunitz/xz v0.5.6 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
 	k8s.io/api v0.0.0-20191025225708-5524a3672fbb
 	k8s.io/apimachinery v0.0.0-20191025225532-af6325b3a843
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-aggregator v0.0.0-20191025230902-aa872b06629d
+	k8s.io/client-go v10.0.0+incompatible
```

Comment:
`client-go` kept coming back when I ran `go mod tidy`, so I guess something is pulling it in? But I forced the right version in the `replace` statement below.